ProjectLighthouse/Controllers/Website/SlotPageController.cs

```diff
@@ -23,6 +23,32 @@ public SlotPageController(Database database)
         this.database = database;
     }
 
+    [HttpGet("rateComment")]
+    public async Task<IActionResult> RateComment([FromRoute] int id, [FromQuery] int commentId, [FromQuery] int rating)
+    {
+        User? user = this.database.UserFromWebRequest(this.Request);
+        if (user == null) return this.Redirect("~/login");
+
+        await this.database.RateComment(user,
+            commentId,
+            rating);
```

Comment:
What happened here? Did our formatting rules screw this up or was this done manually?

Reply:
probably got formatted and I didn't notice it